config/environments/test.rb

```diff
@@ -6,6 +6,7 @@
   # your test database is "scratch space" for the test suite and is wiped
   # and recreated between test runs. Don't rely on the data there!
   config.cache_classes = true
+  config.cache_store = :null_store
```

Comment:
This means that it doesn't cache in test environments (to avoid any cross-test pollution).